### PR TITLE
Update intro-to-css.md

### PR DESCRIPTION
**Pull request by abhilash15500**
A warning would be very helpful as i see people in discord everyday asking doubts about animation exercise. They solve 1to5 exercises and then in flow they start animation even when its instructed. I think a warning should make them aware to not touch anything else except foundations 1 to 5 exercises.

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by XAJX179** [comment]
hello xplozion! just a heads up remove the blank lines at line numbers - 364 , 373 to avoid markdown lint errors and following style guide.

Now on topic -
 i think we should add some writing about reading instructions more carefully in future too, if we add this warning.

**Comment by MaoShizhong** [comment]
Thanks for the PR. Honestly, I'm not too sure this will change much. The assignment very clearly already states to go to the foundations folder and complete the listed exercises. People have generally only done differently when they get carried away and/or just straight up forget the instructions after reading the README but don't go back to check them (so they assume).

In those cases, given the explicit instructions they read to get to that repo had been forgotten, I'm not convinced a note box repeating those instructions beforehand will change this. I'll see what other maintainers have to say though, if any have differing opinions.

I will point out though that [your note box doesn't follow the correct syntax](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#note-boxes). The heading should be a level 4 heading placed inside the div, and should also ideally be more descriptive than a generic "warning".

**Comment by Eduardo06sp** [comment]
@MaoShizhong Is it okay to openly discuss some potential solution to this problem here? 

I share the same concern as the author and have seen multiple users (in the Discord) accidentally go to the more advanced exercises.

**Comment by abhilash15500** [comment]
One solution i can think is directly redirecting them to foundations folder instead of css exercises? That would be better I guess

**Comment by MaoShizhong** [comment]
@Eduardo06sp Please do fire away. If there is a solution, I'm not sure a note box would be it but perhaps there's another solution that's suitable.

And @abhilash15500 we need people to read the README first (and even when we say it clear and link to it, some people still don't read it and we can't really help prevent those cases).

Perhaps we could split the assignment into 2 steps. The first one instructs to read the repo README, linking as it currently does. The second step then links to the Foundations folder and instructs to do exercises 1-5 as it currently does. I'd love to hear what Eduardo wants to say as well

**Comment by abhilash15500** [comment]
Splitting assignment would work nicely. 
Also mb for not raising issue first i was unaware of the workflow for this. 
Just shared what i felt like is an issue.

I would be happy with whatever steps TOP management and core members take for this 😄

**Comment by MaoShizhong** [comment]
Yes ideally an issue is created first so the proposal can be made (examples if appropriate) then it can be discussed, just in case things either don't get approved or perhaps a different solution is more appropriate through discussion, especially if you haven't had a lot of experience with contributing here. Of course if it's a super obvious fix like a typo or broken link, then by all means go straight to a PR if there isn't already an open issue/PR for it.

**Comment by abhilash15500** [comment]
I will remember it next time. 

Im happy to follow whatever steps needed if any from my side / or any decision which top members take which they think its for the betterment of TOP! :)

**Comment by Eduardo06sp** [comment]
@MaoShizhong I think that small change would help a lot.

I also believe that structuring the `css-exercises` repo like the curriculum is structured would serve as another "fail-safe".

I.e.
Instead of the folders being laid out like they currently are
```bash
animation  flex  foundations  grid  margin-and-padding
```

we can structure them just like they appear in the curriculum
```bash
foundations  intermediate-html-and-css  advanced-html-and-css
```

and the nesting can go deeper if deemed necessary,
but the drawback is that any potential, future updates to the curriculum's structure will require an update in the `css-exercises` repo as well

just an idea :}

**Comment by MaoShizhong** [comment]
Thanks @Eduardo06sp. Could you open an issue about that in the [css-exercises repo](https://github.com/TheOdinProject/css-exercises) please? I think that's something better proposed and discussed there (and I'd like input from other team members), and if it gets approved, there will also be a sibling PR here to amend any lessons to reference the right folders and files etc.

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Regarding this PR, @abhilash15500, could you remove the note box changes you originally made then do the following?

- Split the assignment step into 2 steps:
  1. Link to the CSS exercises repo and instruct to read the README
  2. Link to the foundations directory and carry on with the rest of the instructions.

File: foundations/html_css/css-foundations/intro-to-css.md [hunk 1]
```diff
@@ -365,7 +365,8 @@ If you need to add a *unique* style for a *single* element, this method can work
 
 <div class="lesson-content__panel" markdown="1">
 
-1. Go to our [CSS exercises repository](https://github.com/TheOdinProject/css-exercises) and read the README. Then, once you know how to use the exercises, navigate to the `foundations` directory. Review each README file prior to completing the following exercises in order:
+1. Go to our [CSS exercises repository](https://github.com/TheOdinProject/css-exercises) and read the README file.
+1. Then, once you know how to use the exercises, navigate to the [foundations](https://github.com/TheOdinProject/css-exercises/tree/main/foundations) directory. Review each README file prior to completing the following exercises in order:
```
[the remark this answers is not in the document]
Comment (MaoShizhong) [reply foundations/html_css/css-foundations/intro-to-css.md:369]: Also just for future reference, if someone has left a review, they'll appear in the top right corner. Once you've addressed any requested changes, you can hit the rerequest review button to indicate it's ready to be rereviewed.

Screenshot_2024-09-09-13-31-35-19_e4424258c8b8649f6e67d283a50a2cbc

**Review by MaoShizhong — APPROVED** [review]
:pray:

**Comment by abhilash15500** [comment]
Appreciated! ☺️

**Comment by Eduardo06sp** [comment]
Relevant Issue: https://github.com/TheOdinProject/css-exercises/issues/593

**Pull request merged**